mail_private/static/description/index.html

```diff
@@ -0,0 +1,51 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Mail Private</h2>
+            <h3 class="oe_slogan">Send messages to recipients you specified</h3>
```

Comment:
Send private messages to specified recipients, regardless of who are in followers list.